### PR TITLE
LPD-42552 Add check permission for fix a test that is failing

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-42552

Resent https://github.com/brianchandotcom/liferay-portal/pull/157047

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:report:93126186

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/java/com/liferay/commerce/product/asset/categories/navigation/web/internal/display/context/CPAssetCategoriesNavigationDisplayContext.java [hunk 1]
```diff
@@ -139,12 +140,19 @@ public AssetVocabulary getAssetVocabulary() throws PortalException {
...
+					_assetVocabularyService.
+						getAssetVocabularyByExternalReferenceCode(
+							_themeDisplay.getCompanyGroupId(),
+							assetVocabularyExternalReferenceCode);
+			}
+		}
+		catch (PrincipalException principalException) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(principalException);
+			}
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/java/com/liferay/commerce/product/asset/categories/navigation/web/internal/display/context/CPAssetCategoriesNavigationDisplayContext.java:155]: Buy space

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

**Review by alessiorendina — APPROVED** [review]

**Comment by monaco-fabio** [comment]
ci:report:94324200

**Comment by alessiorendina** [comment]
ci:forward

**Pull request closed without merging**